ang/crmUtil.js

```diff
@@ -19,7 +19,7 @@
       if (_.isObject(entity)) {
         // eval content is locally generated.
         /*jshint -W061 */
-        p = backend(eval('('+angular.toJson(entity)+')'), message);
+        p = backend(eval('('+angular.toJson(entity)+')'), action);
```

Comment:
For someone skimming the function signature, it probably isn't obvious how params change. Maybe include something in the docblock?